### PR TITLE
Learn: Introduction to service mesh using linkerd

### PR DESCRIPTION
**Pull request by Blakelist7**
Signed-off-by: Kanwal Singh <kanwal.88singh@gmail.com>

**Description**
This PR adds the final chapter 8 to Introduction to service mesh using linkerd
so this shall fixes: #1986 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6ba933e062c706d1e5e3aebc312f2e87816dffa0 at: https://60f324d6df12b950f58c6a5d--layer5ng.netlify.app

**Comment by Blakelist7** [comment]
@Nikhil-Ladha

File: content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/fault-injection.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  Using SMI Traffic Split API to inject errors
+</h3>
+
+We can easily inject application failures by using the Traffic Split API of the Service Mesh Interface. This allows us to do failure injection in a way that is implementation agnostic and works across service meshes.
+
+We will do this first by deploying a new service which only return errored responses. We will be using a simple NGINX service which has configured to only return HTTP 500 responses.
+
+We will hten create a traffic split which would redirect the service mesh to send a sample percentage of traffic to the error service instead, let's say 20% of service's traffic to error, then we would have injected an artificial 20% error rate in service.
+
+<h4 class="chapter-sub-heading">
```
Comment (Nikhil-Ladha) [inline content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/fault-injection.mdx:27]: Make this `h3`

Comment (Blakelist7) [reply content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/fault-injection.mdx]: done

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**